server/sonar-web/src/main/js/app/components/nav/global/GlobalNavUser.js

```diff
@@ -54,9 +54,9 @@ class GlobalNavUser extends React.PureComponent {
     const { currentUser } = this.props;
     return (
       <li className="dropdown js-user-authenticated">
-        <a className="dropdown-toggle" data-toggle="dropdown" href="#">
-          <Avatar email={currentUser.email} size={20} />&nbsp;
-          {currentUser.name}&nbsp;<i className="icon-dropdown" />
+        <a className="dropdown-toggle navbar-avatar" data-toggle="dropdown" href="#">
+          <Avatar email={currentUser.email} name={currentUser.name} size={24} />
+          <i className="icon-dropdown" />
```

Comment:
It's not rendering well right now, the dropdown icon is misplaced : 
![image](https://cloud.githubusercontent.com/assets/1725611/25845870/e94d8bac-34af-11e7-99d0-cc92a2036290.png)